.vscode/extensions.json.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+
+SPDX-License-Identifier: MPL-2.0
```

Comment:
please add a whilteline at the EOF for consistency

Reply:
Included in the latest commit.